src/docfx/Errors.cs

```diff
@@ -4,12 +4,14 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 
 namespace Microsoft.Docs.Build
 {
     internal static class Errors
     {
+#pragma warning disable SA1124 // DoNotUseRegions
```

Comment:
how about we create three files `Errors.cs, Warnings.cs and Info.cs` put them under one folder?